recipes/perl-try-tiny/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set author = "ETHER" %}
+{% set name = "Try-Tiny" %}
+{% set version = "0.28" %}
+{% set sha256 = "f1d166be8aa19942c4504c9111dade7aacb981bc5b3a2a5c5f6019646db8c146" %}
+
+package:
+  name: perl-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://cpan.metacpan.org/authors/id/{{ author[0] }}/{{ author[:2] }}/{{ author }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [ win ]
+  script: perl Makefile.PL && make && make test && make install 
+
+requirements:
+  build:
+    - perl
+    - make
+  run:
+    - perl
+
+test:
+  commands:
+    - perl -V
+    - perl -e 'use {{ name|replace("-","::") }}'
+
+about:
+  home: https://metacpan.org/release/Try-Tiny
+  license: Artistic-2.0
```

Comment:
according to the website http://search.cpan.org/~ether/Try-Tiny-0.28/lib/Try/Tiny.pm this is MIT licensed?

Reply:
Pinging this issue!